my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
This appears to already be addressed by the catch-all rule you have above

Reply:
Yep, that was an oversight. Also, I see from another post below that the margin and padding rules here are redundant b/c of the css reset. I'll remove the redundant style rules from this file.